e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -594,6 +594,114 @@ func (s *IbcEurekaTestSuite) ICS20TransferERC20TokenfromEthereumToCosmosAndBackT
 	}))
 
 	var returnSendTxHash []byte
+	s.Require().True(s.Run("Transfer tokens back from Cosmos chain with invalid destination should fail", func() {
```

Comment:
I think this should probably be a separate test altogether to not make this happy path test too long.